pgoapi/pgoapi.py

```diff
@@ -218,8 +220,15 @@ def heartbeat(self):
                 res['responses']['lng'] = self._posf[1]
                 f.write(json.dumps(res['responses'], indent=2))
 
+            inventory_items = res['responses']['GET_INVENTORY']['inventory_delta']['inventory_items']
+            self.inventory = Player_Inventory(inventory_items)
+
+            for inventory_item in inventory_items:
```

Comment:
these changes are now also in PR #158 since it's hard to make seperate pull-requests that are mergable and these changes here are small. plus i have actually created player.py and player_stats.py there, go check it out :)